lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -15,6 +13,7 @@ Gem::Specification.new do |spec|
 <%- if config[:mit] -%>
   spec.license       = "MIT"
 <%- end -%>
+  spec.required_ruby_version = Gem::Requirement.new(">= 1.9")
```

Comment:
i'd argue adding a required_ruby_version would be usefull as most new gems would not be run checked against older ruby versions. Yes 2.1 is probably a better default.